### DISCUSSION TITLE
TeamBalancer.MaxBalance. #IDEA

### DISCUSSION THREAD

**Discussion by ZhenyaMartynov** [Sokol-Games-GmbH/PixelCombats.PublicApi#139] [Ideas]
Сделать свойство MaxBalance, будет отвечать за нужное количество игроков в команде, для перекидывания в другую.
Например в 1 команде 10 игроков, а в другой 7, и из команды где 10 только перекинет во 2 команду, надеюсь понятно описал

**Comment by forwork10** [comment]
Вижу вы не знакомы с игровым API... Такая вещь уже существует, MaxPlayerDifference называется... посмотрите апи

**Comment by ZhenyaMartynov** [comment]
Да?) А теперь посмотри внимательно, есть ли у этого контекст

**Comment by forwork10** [comment]
А зачем вам контекст... вы там по 10 команд создает что-ли? Ну можно и двумя обойтись - там всё легко настраивается без проблем

**Comment by ZhenyaMartynov** [comment]
Да, я хочу 10 команд, и? Как ты решишь это

**Comment by forwork10** [comment]
Ну мы не можем под все ваши идеи игру переделывать...

**Comment by kkohno** [comment]
> Сделать свойство MaxBalance, будет отвечать за нужное количество игроков в команде, для перекидывания в другую. Например в 1 команде 10 игроков, а в другой 7, и из команды где 10 только перекинет во 2 команду, надеюсь понятно описал

как писали выше - такое имеется уже (MaxPlayerDifference ) и работает хоть для 10 команд. Идея функционала в том, чтобы команды заполнялись равномерным количеством игроком. ставите 2 например - значит в командах макс разница будет 2 игрока
